Common/cpp/ReanimatedRuntime/WorkletRuntime.h

```diff
@@ -26,18 +28,21 @@ class WorkletRuntime : public jsi::HostObject,
       const std::shared_ptr<MessageQueueThread> &jsQueue,
       const std::shared_ptr<JSScheduler> &jsScheduler,
       const std::string &name,
+      WorkletRuntimeType type,
       const std::string &valueUnpackerCode);
 
   jsi::Runtime &getJSIRuntime() const {
     return *runtime_;
   }
 
+  std::unique_lock<std::recursive_mutex> lock();
```

Comment:
Let's rename this function to `createLock`. I would assume that `lock` would actually lock something and I would look for `unlock`.